src/scattdata.cpp

```diff
@@ -36,6 +37,13 @@ ScattData::base_init(int order, const xt::xtensor<int, 1>& in_gmin,
     energy[gin] = in_energy[gin];
     mult[gin] = in_mult[gin];
 
+    // Make sure the multiplicity does not have 0s
+    for (int go = 0; go < mult[gin].size(); go++) {
+      if (mult[gin][go] == 0.) {
```

Comment:
What do you think about adding a warning there? It's uncommon having a zero multiplicity could almost only be deliberate